datafusion/sql/tests/integration_test.rs

```diff
@@ -2308,6 +2314,22 @@ fn select_multibyte_column() {
     quick_test(sql, expected);
 }
 
+#[test]
+fn select_groupby_orderby() {
+    let sql = r#"SELECT
+  avg(age) AS "value",
+  date_trunc('month', birth_date) AS "birth_date"
+  FROM person GROUP BY birth_date ORDER BY birth_date;
```

Comment:
I suggest add more test about qualified column, I think it may cause some problem.

such as use `date_trunc('month', person.birth_date) AS "birth_date"`
instead of 
`date_trunc('month', birth_date) AS "birth_date"`